src/pages/researcher_console/DatasetSubmissionsTable.jsx

```diff
@@ -134,6 +134,7 @@ export default function DatasetSubmissionsTable(props) {
                 className="glyphicon glyphicon-trash"
                 onClick={() => handleClick(term)}
                 to="#"
+                aria-label={`Delete set '${selectedTerm.datasetIdentifier}'`}
```

Comment:
This link is operating on `term` ... is there a reason we're showing the identifier for `selectedTerm` instead? I do see that we're using `selectedTerm` in the following `ConfirmationDialog` block, but that might be a bug, can you confirm that we're using the correct variable in this div?
```suggestion
                aria-label={`Delete dataset '${term.datasetIdentifier}'`}
```

Reply:
‸‸‸‸. (agree with Greg's point about using `term` instead of `selectedTerm`)

Reply:
@rushtong and @otchet-broad - updating to reflect suggested changes